lib/action_handler.dart

```diff
@@ -45,31 +45,17 @@ class ActionHandler {
     List<Message> messages = <Message>[];
 
     // Check for URLs
-    RegExpMatch? linkMatch;
-    String? linkMsg;
-    List<RegExpMatch> matches = parseLinks(text);
-
-    // Get the first match (if it exists)
-    if (matches.length > 0) {
-      linkMatch = matches.first;
-      linkMsg = text.substring(linkMatch.start, linkMatch.end).trim();
+    int index = text.indexOf("https://");
```

Comment:
What if the link is a `www.` link

Reply:
AM doesn't split it so I didn't add it lol but here's some examples:

`testhttps://google.com` -> `test` and `https://google.com` (same for http links)

`testhttps://google.com test` -> `test` and `https://google.com test` (same for http links)

`testwww.google.com` stays the same (along with any other non http / https links

Reply:
I'll be honest, i don’t want it to be able to pick up `testhttps://google.com test`. To me, that should be an invalid link and not be "preview-able". IMO, links should either be considered links if the following is present:

- Line starts with https://, http://, or, www
- a space, followed by https://, http://, or, www.

I don’t think we should support when it's just a substring

Reply:
Alright so I'll get rid of the link splitting entirely as well as the detection algo for improperly formatted links and we should be good

Reply:
@zlshames this should be good now